### PR TITLE
[Security] az security pricing: Change help to reflect current API version being called

### PR DESCRIPTION
**Pull request by chshum**
**Description**  
Update az security pricing help with correct documentation

**Testing Guide**  
N/A

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/azure-cli/azure/cli/command_modules/security/custom.py [hunk 1]
```diff
@@ -253,22 +253,15 @@ def list_security_pricings(client):
...
-    if resource_group_name:
-        return client.get_resource_group_pricing(resource_group_name, resource_name)
+def get_security_pricing(client, resource_name):
 
     return client.get(resource_name)
 
 
-def create_security_pricing(client, resource_name, tier, resource_group_name=None):
-
-    if resource_group_name:
```
Comment (haroldrandom) [inline src/azure-cli/azure/cli/command_modules/security/custom.py:266]: Will this deletion likely to break user?

Comment (chshum) [reply src/azure-cli/azure/cli/command_modules/security/custom.py:266]: AFAIk this option isnt available in the new api, im not sure if this is auto-generated or not. 
I was unable to reach this code using CLI . tried running commands 
az security pricing create with parameter "resourceGroup" but received error - unknown arguemnt
Is there a way to trace back how this is called and what the parameter name for this is?

image

Comment (haroldrandom) [reply src/azure-cli/azure/cli/command_modules/security/custom.py:266]: you could type --debug to get more verbose output to help.  If you want to see what options currently  there it is, you could run it with -h.  
If some IDEs you are using for debugging ,  it depends.

Comment (chshum) [reply src/azure-cli/azure/cli/command_modules/security/custom.py:266]: Thanks. 

I might be mistaken, but its possible that this command is already broken. 
When running a command (in master, unchanged branch) with the resource group parameter:

_az security pricing show -n "virtualMachines" -g "resourceGroupName"_

[as per doc: ](https://docs.microsoft.com/en-us/cli/azure/security/pricing?view=azure-cli-latest#az_security_pricing_create-optional-parameters)

I get the following error which corresponds to the method being removed:

"CLIInternalError: The command failed with an unexpected error. Here is the traceback:
'PricingsOperations' object has no attribute 'get_resource_group_pricing'"

Comment (haroldrandom) [reply src/azure-cli/azure/cli/command_modules/security/custom.py:266]: I am okay if you are confirmed about it.

**Review by haroldrandom — APPROVED** [review]

**Pull request merged**